core/src/env.rs

```diff
@@ -157,6 +157,9 @@ pub struct LaterExt<E: Ext> {
     inner: Rc<RefCell<Option<E>>>,
 }
 
+unsafe impl<E: Ext> Sync for LaterExt<E> {}
+unsafe impl<E: Ext> Send for LaterExt<E> {}
```

Comment:
I think no, because we have no reasons to use ext in multithread context. I have added this implementations, just because wasmtime demanded them. So, may be comment will be enough?

Reply:
A, forget to mention that by default wasmtime does not use it in multithread context. It can use it, but in our case does not. So, I think that it's some sort of user interface problem in wasmtime

Reply:
Is there a possibility that any unexpected problems arise because of using not thread-safe inner types in `LaterExt`? As I understand your comments, they can't, but can we secure that on type level? Would be nice,

Reply:
Ok, change to Arc